test/stripe/core_resources/payout_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule Stripe.PayoutTest do
+  use Stripe.StripeCase, async: true
+
+  describe "create/2" do
+    test "creates a card for a customer" do
+      assert {:ok, _} = Stripe.Payout.create(%{amount: 100, currency: "USD", source_type: "card"})
```

Comment:
@joshsmith why does it return a map instead of a struct?

Reply:
Means we're not doing something quite right.

Reply:
Did you check to see if the `Converter` has `payout` listed? My guess is that it probably does not.

Reply:
@joshsmith ah good idea.  Fixed.